app/controllers/questions_controller.rb

```diff
@@ -101,6 +101,15 @@ def search
       @questions = Question.search(question_params[:search])
     end
 
+    def report_abuse
```

Comment:
@maskedXchaotic we need one more check that author should not mark his own content as abuse